frontend/src/components/MetricCard/metricCard.css

```diff
@@ -1,10 +1,18 @@
 .metric-card {
-    width: 309px;
-    background: var(--header-button-bg);
-    border: var(--midnight-purple-border);
-    border-radius: 900px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-  }
\ No newline at end of file
+  width: 309px;
+  background: var(--header-button-bg);
+  border: var(--midnight-purple-border);
+  border-radius: 900px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+@media screen and (max-width: 768px) {
+  .metric-card {
```

Comment:
inner padding and title's font-size are not the same as in figma

Reply:
I believe paddings are still incorrect, because in figma this card has smaller height (of 88px)